### ISSUE TITLE
target for Locator?

### ISSUE THREAD

**Issue by pangsg** [Yale-LILY/QMSum#10]
Hi， I'm very interested in your Locator and trying to reproduce the model.
However, there are still some questions that confused me. As the paper says, the Locator is trained by binary cross-entropy loss, however, I can't find the target or ground truth for Locator in this repo. Could you please provide me with some help, thanks a lot!

**Comment by My-timer** [comment]
I have the same question!

**Comment by zhanglwr** [comment]
I have the same question!

> I have the same question!
